tests/serialization_test.py

```diff
@@ -11,7 +11,7 @@ def setUp(self):
         self.maxDiff = None
 
     def test_pickle_serialize_unserialize(self):
-        w = world_gen("Dummy", 32, 16, 1, step=Step.get_by_name("full"))
+        w = world_gen("Dummy", 32, 16, 1, [.874, .765, .594, .439, .366, .124], [.941, .778, .507, .236, 0.073, .014, .002], step=Step.get_by_name("full"))
```

Comment:
Might it be prettier to give world_gen default values for the new parameters so you don't have to explicitly pass them here? Should these values change (or a 7th be added) during later development, it would be a bit ugly to have to go around and fix the tests (instead of e.g. a standard parameter within world_gen).